task/02-numbers-tasks.js

```diff
@@ -54,7 +54,14 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+    // no overflow if inputs' signs differ
+    if (value1 < 0 ^ value2 < 0)
```

Comment:
Really interesting! Please try to solve it easier

Reply:
I thought that I could avoid doing `return value1 / 2 + value2 / 2` so that I'd use only one division operation instead of two, but it seems like this solution ended up looking unnecessarily complicated.